app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,54 @@
+class OrdersController < ApplicationController
+  before_action :find_order, only: [:update, :show, :destroy]
```

Comment:
I really like the way you've used the `before_action` and the only to be sure it is only executed for the specific actions you need